gensim/models/ldamodel.py

```diff
@@ -594,9 +594,19 @@ def __str__(self):
             self.num_terms, self.num_topics, self.decay, self.chunksize
         )
 
-    def sync_state(self):
-        """Propagate the states topic probabilities to the inner object's attribute."""
-        self.expElogbeta = np.exp(self.state.get_Elogbeta())
+    def sync_state(self, current_Elogbeta=None):
+        """Propagate the states topic probabilities to the inner object's attribute.
+
+        Parameters
+        ----------
+        current_Elogbeta: numpy.ndarray
+            Posterior probabilities for each topic, optional.
+            If omitted, it will get Elogbeta from state.
+        """
+
+        if current_Elogbeta is None:
```

Comment:
when this used?

Reply:
It's used In `gensim.models.wrappers.ldamallet.malletmodel2ldamodel`, `gensim.models.lda_worker.reset`
+ to save backward compatibility.
Of course, I can fix this places, but should I?